general/1_terminal/anatolii-kravchenko/task-1.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# This script is intended for create a bunch of files using touch. 
+# Output the names of those that have zoola in their names.
+
+qtyOfFiles=4                            # Set quantity of files that will be created
+
+for (( i = 0; i < $qtyOfFiles; i++ ))   # Loop for creating each file
+do
+    touch "Zoola-$i.txt"
+done
+
+ls -l | grep Zoola                      # Output files where "Zoola" word is present in the filename
```

Comment:
Since we are creating files here, let's add a cleanup section, where we would delete everything that we have created.

Reply:
I added this feature in last commit.